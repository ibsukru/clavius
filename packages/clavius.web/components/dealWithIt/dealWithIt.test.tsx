import { shallow } from 'enzyme'
import React from 'react'
import DealWithIt from './dealWithIt'

test('DealWithIt', () => {
  const wrapper = shallow(<DealWithIt text="Hello world" />)
  expect(wrapper).toMatchSnapshot()
})
