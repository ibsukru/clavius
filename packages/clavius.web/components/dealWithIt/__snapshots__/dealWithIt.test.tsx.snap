// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`DealWithIt 1`] = `
<span>
  Hello world (▀̿Ĺ̯▀̿ ̿).
</span>
`;
